src/__tests__/develop.spec.js

```diff
@@ -0,0 +1,20 @@
+import GetNumberSum from '../develop.js'
+
+function test() {
+    const testCases = [
+      { input: { a: 1, b: 2 }, expected: 3 },
+      { input: { a: -2, b: 1 }, expected: 0 },
+      { input: { a: 'not a number', b: 'number' }, expected: NaN } 
```

Comment:
We are still expecting wrong output
Just run out before push.
You will recive `[true, true, false]`
If we reciving `false` this means that we failure our test and func's logic is incorrect
But for our case its just an mistake.

Reply:
We are testing your function 
There is no need to use chatGPT. 
I see that this is genereted by him. Please, try to write your own code to make yourself better.
ChatGPT cannot write hole program and make your code much worst. :name_badge: